hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionPolicy.java

```diff
@@ -15,6 +15,7 @@
     private static final int DEFAULT_MESSAGE_TTL = 3600;
     private static final int DEFAULT_MESSAGE_BACKOFF = 100;
     private static final int DEFAULT_REQUEST_TIMEOUT = 1000;
+    private static final int DEFAULT_SOCKET_TIMEOUT = 0;
```

Comment:
When you set it to 0 it will be no socket timeout at all

Reply:
So currently created subscriptions are unaffected. With this default option, only subscription's request timeout will be respected